libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -147,6 +147,8 @@ public AztecLoggingException(Throwable originalException) {
     private static final String TEMP_VIDEO_UPLOADING_CLASS = "data-temp-aztec-video";
     private static final String GUTENBERG_BLOCK_START = "<!-- wp:";
 
+    private static final String AUTHORIZATION_HEADER_NAME = "Authorization";
```

Comment:
Would it make sense to reuse the `AUTHORIZATION_HEADER_NAME` from `AuthenticationUtils` here instead of declaring a new variable?

Reply:
Yes, but `AztecEditorFragment` is part of the Editor submodule, so It has no access to `AuthenticationUtils`. This is not super elegant, but my understanding is that we slowly but surely switch towards block editor, so I decided not to add any new logic related to it. But maybe there is some simple way to fix it, that I'm missing?

Reply:
> Yes, but `AztecEditorFragment` is part of the Editor submodule, so It has no access to `AuthenticationUtils`. 

Right, I forgot about that minor detail. 🤦‍♂